python/pyspark/sql/functions.py

```diff
@@ -1846,6 +1846,27 @@ def array_contains(col, value):
     return Column(sc._jvm.functions.array_contains(_to_java_column(col), value))
 
 
+@since(2.4)
```

Comment:
We need to annotate as `@ignore_unicode_prefix`?

Reply:
I see, thanks